network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -1,23 +1,25 @@
 {% extends "pages/base.html" %}
 
-{% load bg_nav_tags %}
+{% load bg_nav_tags localization i18n %}
 
+{% get_current_language as lang_code %}
 
-{% block page_title %}{% if pageTitle %}{{ pageTitle }}{% else %}Mozilla - *privacy not included{% endif %}{% endblock %}
+{% block page_title %}{% if pageTitle %}{{ pageTitle }}{% else %}Mozilla - {% blocktrans context "This can be localized. This is a reference to the “*batteries not included” mention on toys." %}*privacy not included{% endblocktrans %}{% endif %}{% endblock %}
```

Comment:
I didn't know this was true - can we get @caltrider's signoff on this one?

Reply:
Got a sign-off over Slack (Thanks Jen!)